### PR TITLE
[Task#68272] Add view registrations UI for Admin

### PR DESCRIPTION
**Pull request by tungbd-0214**
## Related Tickets:
- [#68272](https://edu-redmine.sun-asterisk.vn/issues/68281)

## WHAT this PR does?
- Add view registrations UI for Admin

## Notes Impacted Areas:
- N/A.

## Attachments:

Screen Shot 2023-09-13 at 21 19 27

Screen Shot 2023-09-14 at 14 19 35

### CODE REVIEW

**Review by Hungnd-0370 — APPROVED** [review]

**Comment by tungbd-0214** [comment]
ready

File: resources/js/Pages/Registrations.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    {
+        title: 'Registration Package',
+        dataIndex: 'registration_package',
+    },
+    {
+        title: 'Action',
+        render: (text, record) => {
+            return (
+                <div>
+                    <Button className="hover:bg-slate-300 hover:text-gray-950">Accept</Button>
```
Comment (ghost) [inline resources/js/Pages/Registrations.jsx]: localization

File: resources/js/Pages/Registrations.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    },
+];
+
+
+export default function Registrations(props) {
+
+    const [registrations, setRegistrations] = useState(props[0].registrations);
+
+    const onTableChange = (pagination, filters, sorter, extra) => {
+        console.log('params', pagination, filters, sorter, extra);
```
Comment (ghost) [inline resources/js/Pages/Registrations.jsx]: ko console.log()

File: resources/js/Pages/Registrations.jsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    };
+
+    console.log(props[0].registrations)
```
Comment (ghost) [inline resources/js/Pages/Registrations.jsx]: ko console.log

File: resources/js/Pages/Registrations.jsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    return (
+        <Authenticated
+            auth={props.auth}
+            errors={props.errors}
+            header={<h2 className="font-semibold text-xl text-gray-800 leading-tight">Registrations</h2>}
```
Comment (ghost) [inline resources/js/Pages/Registrations.jsx]: localization

File: routes/web.php [hunk 5]
```diff
@@ -30,6 +31,15 @@ function () {
     }
 );
 
+Route::get(
+    '/registrations',
+    function () {
+        return Inertia::render('Registrations', [ [
+            'registrations' => Registration::all(),
+        ]]);
+    }
```
Comment (ghost) [inline routes/web.php]: ko code trong web.php

**Comment by tungbd-0214** [comment]
ready

**Pull request merged**